src/content/docs/backend/API/unit-tutorial_streams.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Doubtfire API
+---
+
+# List of Doubtfire
+
+# API: Units (Page 1)
+
+units : Operations about units
+
+This markdown document provides detailed documentation for the "units" API endpoints, including their URLs, methods, parameters (if any), responses, and example requests using curl.
+
+This "units" API page has the following operations (**_3 operations_**).
```

Comment:
The formatting here does not fit the other pages on the site, please look over this again